From d88911ee23429d665caeaa7401173d81a06ae977 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:26:44 -0400
Subject: [tools] perf/probe: Fix typo

Message-id: <1408462094-14194-236-git-send-email-jolsa@redhat.com>
Patchwork-id: 88282
O-Subject: [PATCH RHEL7.1 BZ1131394 235/325] perf probe: Fix typo
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 9ef0438a957937bf0edc26d58bce891034ff9e30
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Thu Oct 24 17:36:31 2013 -0300

description
===========
s/tyep/type/g.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/probe-finder.c b/tools/perf/util/probe-finder.c
index e41b094..2200dad 100644
--- a/tools/perf/util/probe-finder.c
+++ b/tools/perf/util/probe-finder.c
@@ -583,7 +583,7 @@ static int convert_variable_fields(Dwarf_Die *vr_die, const char *varname,
  }
 
  if (die_find_member(&type, field->name, die_mem) == NULL) {
-  pr_warning("%s(tyep:%s) has no member %s.\n", varname,
+  pr_warning("%s(type:%s) has no member %s.\n", varname,
       dwarf_diename(&type), field->name);
   return -EINVAL;
  }
-- 
1.7.1